keps/sig-node/20190129-hugepages.md

```diff
@@ -552,6 +556,18 @@ locality guarantees as a feature of QoS.  In particular, pods in the
 - E2E testing validating its usage.
 -- https://k8s-testgrid.appspot.com/sig-node-kubelet#node-kubelet-serial&include-filter-by-regex=Feature%3AHugePages
 
+## Test Plan
+
+- A test plan will consist of the following tests
+  - Unit tests
+    - Each unit test for enhancement will be implemented.
+  - E2E tests
+    - There is a test suit for huge pages in e2e test, it will be extended to validate enhancements.
+    - here: https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/hugepages_test.go
+  - cri-tools
+    - Test case will be added to cri-tools to be used in CRI runtime' test(CI).
+    - here: https://github.com/kubernetes-sigs/cri-tools
```

Comment:
How much work will this be?
How much do you think needs to be built to test this kep?

Reply:
This works will be done in the almost same manner of extending e2e-node testsuit of hugepages.
In the cri-tools, It already has testsuit for hugepages.
So, I will add additional test cases on the test suite for both of enhancements.

The mechanism will be:
1. build container cgroup configuration with hugepages config.
(Now hugepages config is set on only pod config with only one size of hugepages)
2. send send container cgroup configuration to CRI runtimes
3. get container creation result from CRI runtime.

Reply:
I think it can be done in 1.19

Reply:
Just for more detail: containerd project uses cri-tools in it's CI.